src/test/java/com/ays/admin_user/service/impl/AdminUserServiceImplTest.java

```diff
@@ -0,0 +1,100 @@
+        // Given
+        AdminUserListRequest mockAdminUserListRequest = new AdminUserListRequestBuilder().withValidValues().build();
+
+        List<AdminUserEntity> mockAdminUserEntities = Collections.singletonList(new AdminUserEntityBuilder().build());
+        Page<AdminUserEntity> mockPageAdminUserEntities = new PageImpl<>(mockAdminUserEntities);
+
+        List<AdminUser> mockAdminUsers = ADMIN_ENTITY_TO_ADMIN_MAPPER.map(mockAdminUserEntities);
+        AysPage<AdminUser> mockAysPageAdminUsers = AysPage.of(mockPageAdminUserEntities, mockAdminUsers);
+
+        AysUserType aysUserType = AysUserType.SUPER_ADMIN;
+
+        // When
+        Mockito.when(aysIdentity.getUserType())
+                .thenReturn(aysUserType);
+        Mockito.when(adminUserRepository.findAll(mockAdminUserListRequest.toPageable()))
+                .thenReturn(mockPageAdminUserEntities);
+
+        // Then
+        AysPage<AdminUser> aysPageAdminUsers = adminUserService.getAdminUsers(mockAdminUserListRequest);
+
+        AysPageBuilder.assertEquals(mockAysPageAdminUsers, aysPageAdminUsers);
+
+        Mockito.verify(adminUserRepository, Mockito.times(1))
+                .findAll(mockAdminUserListRequest.toPageable());
+    }
+
+    @Test
+    void givenUserListRequest_whenAdminwithRoleIsAdmin_thenReturnAllAdminUsers() {
+        // Given
+        AdminUserListRequest mockAdminUserListRequest = new AdminUserListRequestBuilder().withValidValues().build();
+
+        AdminUserEntity mockAdminUserEntity = new AdminUserEntityBuilder().withValidFields().build();
+
+        List<AdminUserEntity> mockAdminUserEntities = Collections.singletonList(mockAdminUserEntity);
+        Page<AdminUserEntity> mockPageAdminUserEntities = new PageImpl<>(mockAdminUserEntities);
+
+        List<AdminUser> mockAdminUsers = ADMIN_ENTITY_TO_ADMIN_MAPPER.map(mockAdminUserEntities);
+        AysPage<AdminUser> mockAysPageAdminUsers = AysPage.of(mockPageAdminUserEntities, mockAdminUsers);
+
+        AysUserType aysUserType = AysUserType.ADMIN;
+        Specification<AdminUserEntity> specification = (root, query, criteriaBuilder) -> criteriaBuilder
+                .equal(root.get("organizationId"), mockAdminUserEntity.getOrganizationId());
+
+        // When
+        Mockito.when(aysIdentity.getUserType()).thenReturn(aysUserType);
+        Mockito.when(aysIdentity.getOrganizationId()).thenReturn(mockAdminUserEntity.getOrganizationId());
+
+        Mockito.when(adminUserRepository.findAll(Mockito.any(Specification.class), Mockito.any(Pageable.class)))
+                .thenReturn(mockPageAdminUserEntities);
+
+        AysPage<AdminUser> aysPageAdminUsers = adminUserService.getAdminUsers(mockAdminUserListRequest);
+
+        // Then
+        AysPageBuilder.assertEquals(mockAysPageAdminUsers, aysPageAdminUsers);
+
+        Mockito.verify(adminUserRepository, Mockito.times(1))
+                .findAll(Mockito.any(Specification.class), Mockito.any(Pageable.class));
+    }
+}
```

Comment:
In Java, adding a blank line at the end of a class file is recommended for better readability and adherence to coding conventions. It visually separates the class from other code and improves code organization. GitHub may display a warning if the blank line is missing to remind developers to follow this convention and maintain a consistent coding style.

Reply:
Ok. I added one line space at the end of the method and the end of a class file. Can you look through it again?

Reply:
No action has been taken regarding this comment. Can we review it again?

Reply:
No action has been taken regarding this comment. Can we review it again?

Reply:
@Rapter1990 No action has been taken regarding this comment. Can we review it again?